nilearn/plotting/tests/test_surf_plotting.py

```diff
@@ -0,0 +1,313 @@
+                        bg_map=np.random.randn(mesh[0].shape[0]-1,))
+
+    # Wrong size of surface data
+    assert_raises_regex(ValueError,
+                        'surf_map does not have the same number of vertices',
+                        plot_surf, mesh,
+                        surf_map=np.random.random(mesh[0].shape[0]+1,))
+
+    assert_raises_regex(ValueError,
+                        'surf_map can only have one dimension', plot_surf,
+                        mesh, surf_map=np.random.random((mesh[0].shape[0], 2)))
+
+
+def test_plot_surf_stat_map():
+    mesh = _generate_surf()
+    bg = np.random.randn(mesh[0].shape[0],)
+    data = 10*np.random.randn(mesh[0].shape[0],)
+
+    # Plot mesh with stat map
+    plot_surf_stat_map(mesh, stat_map=data)
+    plot_surf_stat_map(mesh, stat_map=data, alpha=1)
+
+    # Plot mesh with background and stat map
+    plot_surf_stat_map(mesh, stat_map=data, bg_map=bg)
+    plot_surf_stat_map(mesh, stat_map=data, bg_map=bg,
+                       bg_on_stat=True, darkness=0.5)
+
+    # Apply threshold
+    plot_surf_stat_map(mesh, stat_map=data, bg_map=bg,
+                       bg_on_stat=True, darkness=0.5,
+                       threshold=0.3)
+
+    # Change vmax
+    plot_surf_stat_map(mesh, stat_map=data, vmax=5)
+
+    # Change colormap
+    plot_surf_stat_map(mesh, stat_map=data, cmap='cubehelix')
+
+    # Save execution time and memory
+    plt.close()
+
+
+def test_plot_surf_stat_map_error():
+    mesh = _generate_surf()
+    bg = np.random.randn(mesh[0].shape[0],)
+    data = 10*np.random.randn(mesh[0].shape[0],)
+
+    # Try to input vmin
+    assert_raises_regex(ValueError,
+                        'this function does not accept a "vmin" argument',
+                        plot_surf_stat_map, mesh, stat_map=data, vmin=0)
+
+    # Wrong size of stat map data
+    assert_raises_regex(ValueError,
+                        'surf_map does not have the same number of vertices',
+                        plot_surf_stat_map, mesh,
+                        stat_map=np.hstack((data, data)))
+
+    assert_raises_regex(ValueError,
+                        'surf_map can only have one dimension',
+                        plot_surf_stat_map, mesh,
+                        stat_map=np.vstack((data, data)).T)
+
+
+def test_plot_surf_roi():
+    mesh = _generate_surf()
+    roi1 = np.random.randint(0, mesh[0].shape[0], size=5)
+    roi2 = np.random.randint(0, mesh[0].shape[0], size=10)
+    parcellation = np.random.random(mesh[0].shape[0])
+
+    # plot roi
+    plot_surf_roi(mesh, roi_map=roi1)
+
+    # plot parcellation
+    plot_surf_roi(mesh, roi_map=parcellation)
+
+    # plot roi list
+    plot_surf_roi(mesh, roi_map=[roi1, roi2])
+
+    # Save execution time and memory
+    plt.close()
+
+
+def test_plot_surf_roi_error():
+    mesh = _generate_surf()
+    roi1 = np.random.randint(0, mesh[0].shape[0], size=5)
+    roi2 = np.random.randint(0, mesh[0].shape[0], size=10)
```

Comment:
Here you are using the global, unseeded PRNG. We try to always avoid this, and to use a local PRNG created by rng = np.random.RandomState(0)